### DISCUSSION TITLE
Is it possible to create an overlay for  barSeries?

### DISCUSSION THREAD

**Discussion by DimiHk** [airbnb/visx#1618] [General]
Basically I'm trying to implement an overlay effect on hover like vercel has, is there any way to do it for barSeries?

this is the example of vercel, something like this.
![image](https://user-images.githubusercontent.com/49434748/209986992-594ab0c6-283b-4055-91dc-1b08e4c45787.png)

this is my example 
![image](https://user-images.githubusercontent.com/49434748/209987203-edfe4abc-869e-4642-ac9f-7fbb8efafb06.png)

**Comment by williaster** [comment]
Hi @DimiHk going to convert this to a discussion since it's not a visx issue.
